<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>train2</class>
 <widget class="QWidget" name="train2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>train2</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>901</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,100);
border-width:4px;
border-radius:0px;
border-color:rbga(255,255,255,30);
color:rgb(204,0,0);</string>
   </property>
   <widget class="QLCDNumber" name="time_LCD">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,100);
border-width:4px;
border-radius:0px;
border-color:rbga(255,255,255,30);
color:rgb(204,0,0);</string>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="intValue" stdset="0">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="home_LCDNumber">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,100);
border-width:4px;
border-radius:0px;
border-color:rbga(255,255,255,30);
color:rgb(204,0,0);</string>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="guest_LCDNumber">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>220</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,100);
border-width:4px;
border-radius:0px;
border-color:rbga(255,255,255,30);
color:rgb(204,0,0);</string>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="GUEST">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>160</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 36pt &quot;Tibetan Machine Uni&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>客   场</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="HOME">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 36pt &quot;Tibetan Machine Uni&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>主   场</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>490</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_button">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>490</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
   </widget>
   <widget class="QLabel" name="QTR">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 36pt &quot;Tibetan Machine Uni&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string> 节</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="qtr_LCDNumber">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>220</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,100);
border-width:4px;
border-radius:0px;
border-color:rbga(255,255,255,30);
color:rgb(204,0,0);</string>
    </property>
    <property name="digitCount">
     <number>1</number>
    </property>
    <property name="intValue" stdset="0">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="return_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>490</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>回正</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="qtr_time_lcd">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>340</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,100);
border-width:4px;
border-radius:0px;
border-color:rbga(255,255,255,30);
color:rgb(204,0,0);</string>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="intValue" stdset="0">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="home_stop_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>剩余暂停：</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="home_stop_num_lcd">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>1</number>
    </property>
    <property name="value" stdset="0">
     <double>5.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="guest_stop_label">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>370</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>剩余暂停：</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="guest_stop_num_lcd">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>370</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>1</number>
    </property>
    <property name="value" stdset="0">
     <double>5.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="home_foul_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>犯规数：</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="home_foul_num_lcd">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>440</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>1</number>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="guest_foul_num_lcd">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>440</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>1</number>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="guest_foul_label">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>440</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>犯规数：</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>241</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,100);
border-width:4px;
border-radius:0px;
border-color:rbga(255,255,255,30);
color:rgb(204,0,0);</string>
   </property>
   <widget class="QPushButton" name="home_stop_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
   </widget>
   <widget class="QToolButton" name="home_add_toolButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>得分</string>
    </property>
    <property name="popupMode">
     <enum>QToolButton::MenuButtonPopup</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="home_dec_toolButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>扣分</string>
    </property>
    <property name="popupMode">
     <enum>QToolButton::MenuButtonPopup</enum>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextOnly</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="home_foul_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>犯规</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>130</y>
     <width>241</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,100);
border-width:4px;
border-radius:0px;
border-color:rbga(255,255,255,30);
color:rgb(204,0,0);</string>
   </property>
   <widget class="QPushButton" name="guest_stop_button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
   </widget>
   <widget class="QToolButton" name="guest_add_toolButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>得分</string>
    </property>
    <property name="popupMode">
     <enum>QToolButton::MenuButtonPopup</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="guest_dec_toolButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>扣分</string>
    </property>
    <property name="popupMode">
     <enum>QToolButton::MenuButtonPopup</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="guest_foul_button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>犯规</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
